### PR TITLE
VBM-131:deep linking feature added

### PR DESCRIPTION
**Pull request by mazurGit**

### CODE REVIEW

File: packages/mobile/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,10 +19,18 @@
         android:launchMode="singleTask"
         android:windowSoftInputMode="adjustResize"
         android:exported="true">
-        <intent-filter>
+        <intent-filter android:autoVerify="true">
```
Comment (AleksandrTermenzhy) [inline packages/mobile/android/app/src/main/AndroidManifest.xml:22]: Don't we need `android:autoVerify="true"` to be within the next `intent-filter`?

File: packages/mobile/src/common/enums/app-linking/app-linking-prefixes.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+enum AppLinkingPrefixes {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/common/enums/app-linking/app-linking-prefixes.ts:1]:
```suggestion
enum AppLinkingPrefix {
```

File: packages/mobile/src/config/app-linking/config.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            },
+          },
+        },
+      },
+    },
+    [RootScreenName.ITEM_INFO]: {
+      path: 'items/:itemId?',
+      parse: {
+        itemId: (itemId: string) => itemId,
+      },
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/config/app-linking/config.ts:24]: Params are parsed as strings **by default**

**Review by AleksandrTermenzhy — APPROVED** [review]
Minor fixes

**Review by VladimirBludov — APPROVED** [review]

File: packages/mobile/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (mazurGit) [reply packages/mobile/android/app/src/main/AndroidManifest.xml:22]: intent
This is an example from documentation

File: packages/mobile/src/config/app-linking/config.ts (hunk 3 above)
Comment (mazurGit) [reply packages/mobile/src/config/app-linking/config.ts:24]: ts_error
I get error from TS if I don't specify  it

**Pull request merged**